### PR TITLE
feat: set client version for integrations

### PR DESCRIPTION
**Pull request by mzanoni**
https://trello.com/c/ksgmWgad

### CODE REVIEW

File: packages/integrations/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "publish": "npm run build && npm run build:types"
   },
   "peerDependencies": {
-    "@relewise/client": "^2.0.0"
+    "@relewise/client": ">=1.0.0 <3.0.0"
```
Comment (mzanoni) [inline packages/integrations/package.json:37]:
```suggestion
    "@relewise/client": ">=1.22.0 <3.0.0"
```

File: packages/integrations/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "publish": "npm run build && npm run build:types"
   },
   "peerDependencies": {
-    "@relewise/client": "^2.0.0"
+    "@relewise/client": ">=1.22.0 <3.0.0"
```
Comment (mzanoni) [inline packages/integrations/package.json:37]: In the latest release this is how this is configured:
`"@relewise/client": "^1.22.0"`

This should allow this to be runned with any version between 1.22.0 and 3.0.0, where `3.0.0` and above is excluded

Comment (SWH-Relewise) [reply packages/integrations/package.json:37]: Cool! 👏

**Review by SWH-Relewise — APPROVED** [review]
Awesome!

**Pull request merged**